tasks.py

```diff
@@ -426,29 +435,21 @@ def unittest_coverage(context):
     run_command(context, command)
 
 
-@task
-def integration_tests(context):
-    """Some very generic high level integration tests."""
-    session = requests.Session()
-    retries = 1
-    max_retries = 60
-
-    start(context)
-    while retries < max_retries:
-        try:
-            response = session.get("http://localhost:8080", timeout=5)
-        except requests.exceptions.ConnectionError:
-            print("Nautobot not ready yet sleeping for 5 seconds...")
-            sleep(5)
-            retries += 1
-            continue
-        if response.ok:
-            print("Nautobot is ready...")
-            break
-        else:
-            raise Exit(f"Nautobot returned and invalid status {response.status_code}", 1)
-    if retries >= max_retries:
-        raise Exit("Timed out waiting for Nautobot", 1)
+@task(
+    help={
+        "keepdb": "save and re-use test database between test runs for faster re-testing.",
+        "label": "specify a directory or module to test instead of running all Nautobot tests",
+        "failfast": "fail as soon as a single test fails don't run the entire test suite",
+        "buffer": "Discard output from passing tests",
+        "verbose": "Toggle verbose test output",
+    }
+)
+def integration_test(
```

Comment:
Does it make sense to add the `integration_test` task to the `tests` wrapper task below?

Reply:
I would say no for now, only because this is now moving us towards the hybrid Docker/localhost pattern since Selenium is only instrumented with containers, and I don't think we are quite ready to start refactoring the documentation to account for that just yet.

On that note, I am going to update the developer docs right meow to at least describe the steps.

Reply:
Fair enough. Maybe at least update the description of the `tests` task to something like "Run all *unit* tests and linters" in the meantime?